### ISSUE TITLE
Run Gradio Log With Loguru

### ISSUE THREAD

**Issue by sglbl** [louis-she/gradio-log#6]
Loguru is a really popular library (20k+ stars) and many people use it because of its simplicity. I wanted to use that with `gradio_log` but even though it writes to the file totally fine, it creates duplicate lines on user interface: 
Minimal reproducible example:
```python
import sys
import time
import gradio as gr
import gradio_log as grl
from loguru import logger
from pathlib import Path


def run_mode():
    for i in range(21):
        logger.debug(f"Running the app {i}")
        time.sleep(1)
    

def set_loguru_logger(level="TRACE", write_to_file=False):
    logger.remove() # remove the default logger
    
    # Format the logger with new settings
    logger.add(sys.stdout, format = "<lvl>{message}</lvl>", colorize=True, backtrace=True, diagnose=True, level=level)  
    if write_to_file:
        # remove the file if it exists
        Path("/tmp/app_logs.log").unlink(missing_ok=True)
        logger.add("/tmp/app_logs.log", format = "<lvl>{message}</lvl>", colorize=True, level=level)


if __name__ == "__main__":
    # Create a logger
    set_loguru_logger("DEBUG", write_to_file=True)
    
    # Run the app
    with gr.Blocks()as demo:
        # Define the different views
        run_mode_button = gr.Button("Run Mode")
        with gr.Row() as logs:
            grl.Log("/tmp/app_logs.log", dark=True, label="Logs of the application")

        run_mode_change = run_mode_button.click(run_mode, inputs=[], outputs=[])       

    demo.queue()

    demo.launch(
        show_error=True,
        show_api=True
    )
```

Screenshots:
![image](https://github.com/user-attachments/assets/36da8f7f-8c46-4d70-bd3f-42181b219697)

My environment:
```bash
(test-py3.10) antioch@hatay:~/test$ pip list | grep 'gradio\|log'
gradio                                  5.6.0
gradio_client                           1.4.3
gradio_log                              0.0.7
loguru                                  0.7.2
```

**Comment by louis-she** [comment]
thanks for the snippet, will have a look at this one ..

https://github.com/louis-she/gradio-log/commit/6f334c2d2d2b509ed7cb00e7ced90c1e47310b56

**Comment by louis-she** [comment]
well... nothing todo with loguru, it's a bug, just released `0.0.8` to fix it.

**Comment by sglbl** [comment]
@louis-she Thank you, it works perfectly. Also I don't need to set colors etc. by myself with the help of loguru
